onnxruntime/core/graph/graph_utils.h

```diff
@@ -378,6 +378,8 @@ bool RemoveNodesWithOneOutputBottomUp(Graph& graph, const Node& node);
  */
 NodeArg& CreateNodeArg(Graph& graph, const NodeArg& base_arg);
 
+bool GetAxesFromUnsqueezeNode(const Graph& graph, const Node& unsqueeze, InlinedVector<int64_t>& axes);
```

Comment:
if it is only used by reshape_fusion, you can put it in reshape_fusion.cc

Reply:
done